packages/longhorn/package.yaml

```diff
@@ -0,0 +1,2 @@
+url: https://github.com/longhorn/charts/releases/download/longhorn-1.0.0/longhorn-1.0.0.tgz
+packageVersion: 00
```

Comment:
You might want to leave `packageVersion` empty since this is directly copied from longhorn charts.